compose/src/main/java/com/maplibre/compose/runtime/nodes/MapCameraNode.kt

```diff
@@ -103,7 +104,9 @@ private class CameraTransitionListener(
 }
 
 private fun cameraUpdate(map: MapboxMap, camera: MapViewCamera) {
-  val cameraUpdate = CameraUpdateFactory.newCameraPosition(camera.toCameraPosition())
+  val cameraPosition = camera.toCameraPosition(map) ?: return
```

Comment:
What's this? Well, it could technically fail (per the C++ API), so we may need to bail. I assume this could only happen for an invalid bbox.